app/middlewares/schema_validation.js

```diff
@@ -0,0 +1,11 @@
+const { checkSchema, validationResult } = require('express-validator/check');
+const errors = require('../errors');
+
+const checkValidationResult = (req, res, next) => {
+  const error = validationResult(req).errors.map(err => err.msg);
+  return validationResult(req).isEmpty() ? next() : next(errors.badRequestError(error));
```

Comment:
here you are passing an array to errors.badRequestError instead of a string :thinking: 
`error` variable is an array